api/sip/client.py

```diff
@@ -235,7 +235,7 @@ class SIPClient(Constants):
     ]
 
     def __init__(self, target_server, target_port, login_user_id=None,
-                 login_password=None, location_code=None, separator=None,
+                 login_password=None, location_code=None, institution_id=None, separator=None,
```

Comment:
Note that institution_id is set to None by default...

Reply:
Ah, I missed the implication of the initial value: None is an appropriate default setting for an optional _field_, but when a _field_ is required, it's default setting must be an appropriate _value_ (''/blank in this case).